### DISCUSSION TITLE
New User Here - Beta apps

### DISCUSSION THREAD

**Discussion by TitusCF** [everysight-maverick/sdk#2] [General]
Hi! So I was most looking forward to some OOB apps I could do some basic testing, how do I access these as they were part of the reason I felt comfortable purchasing these in the first place.
Feeling extremely deceived. Per what was specifically said on the Maverick site the glasses should be a stand alone unit but NOW it appears they require a companion app in order to do anything at all? Where is this companion app? Did I get ripped off?

Also another guy has been here since August and he's the only other? Where is the software for this thing?

**Comment by feranf** [comment]
Hello,

I appreciate your concerns and want to provide some clarity. The Out-of-Box (OOB) beta apps are in their final testing phase and will be released soon. We understand the anticipation and want to assure you that we are working diligently to make it available. 

Kind reminder that the Maverick you've ordered is a **Developer Edition**, specifically designed for developers to create their own applications.The upcoming OOB beta apps aims to help developers to explore various implementation ideas, functionalities, and experiment with potential capabilities of the Maverick.

Regarding the limited software discussions, I guess this is because the units you and others ordered are about to be sent soon. We hope you'll start to see more information and discussions related to the software after you will receive them.

Yes, you understood correctly - that Maverick requires a companion app. It was always like that.

Hope that helped,

If you have any further questions or concerns, feel free to reach out support@everysight.com.

**Comment by feranf** [comment]
Hi Again,
There is a page for the OOB beta apps here: [https://everysight.github.io/maverick_docs/maverick-apps/](https://everysight.github.io/maverick_docs/maverick-apps/)
